addons/rating/views/rating_templates.xml

```diff
@@ -16,6 +16,16 @@
             </t>
         </template>
 
+        <!-- External page: Invalid partner rating -->
+        <template id="invalid_partner_rating_external_page_view" primary="True"
+            inherit_id="rating.rating_external_page_view" name="Not allows to rating">
```

Comment:
I would not inherit the template and instead re-define the whole content.

With your approach, one could customize the block inside "oe_structure_rating_footer_thanks_you" and it would appear on your new template as well, which is not intended.